querydsl-core/src/test/java/com/querydsl/core/support/CollectionAnyVisitorTest.java

```diff
@@ -78,8 +78,5 @@ protected Predicate exists(Context c, Predicate condition) {
         Expression<?> transformed = expression.accept(visitor, new Context());
```

Comment:
Keep in mind, that collectionAnyVisitor is now stateful.
The tests should reflect that as well.